app/web/src/components/ModelingView/ModelingRightClickMenu.vue

```diff
@@ -416,6 +416,18 @@ const rightClickMenuItems = computed(() => {
       disabled,
     });
   }
+  if (
+    restorableSelectedComponents.value.length === 0 &&
+    featureFlagsStore.TEMPLATE_MGMT_FUNC_GENERATION
```

Comment:
Does this also ensure that no view hexagons are selected?

Reply:
Yes! The code on line 298 prevents any of the stuff below that line from having view hexagons selected.